app/templates/base_root.html

```diff
@@ -20,7 +20,7 @@
           <div class="container">
               <div class="navbar-translate">
                   <a class="navbar-brand" href="{{ url_for('index') }}">
-                     App Name
+                     Literapy
```

Comment:
Since you changed this in both branches you will have a merge conflict.

Reply:
EDIT: Just checked, you should be fine to merge. Since the changes are exactly the same, no merge conflicts will arise :-)